.changeset/sharp-snails-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
```

Comment:
Strictly this is not a change to the Wrangler package. It is only changing some fixtures and templates, which are not deployed as part of Wrangler.
I think we could actually dispense with this changeset altogether.